.circleci/config.yml

```diff
@@ -53,6 +54,23 @@ restore-flow-typed-cache: &restore-flow-typed-cache
   restore_cache:
     key: *flow-typed-cache-key
 
+save-assets-cache: &save-assets-cache
+  save_cache:
+    key: *assets-cache-key
+    paths:
+      - public/assets
+      - public/packs-test
+      - tmp/cache/assets
+      - tmp/cache/webpacker
```

Comment:
`tmp` should not be cached ... it is a bad idea ;)
Generally (it might not happen in this case) the directory have the process PID in them so it might be at best case useless, at worse error prone

Reply:
If these dirs are not cached, they are re-created. So we save time by caching them. It is also in the CircleCI example. This is project tmp dir, not `/tmp`.

Reply:
I get that but I think it is riskier because of _random_ PID generation or something we do not really know how sprockets, webpacker do while generating those directories

Reply:
`assets:precompile` should handle whatever is in the cache. This is what other people use. Hope it will be stable. I also don't like that these tools use and require caching of multiple directories that are not even documented (or I didn't see them documented).

Reply:
btw I also see people caching bootsnap https://github.com/jetthoughts/jt_tools/blob/master/lib/install/.circleci/config.yml I wonder if that could benefit us.